packages/rocketchat-message-attachments/client/stylesheets/messageAttachments.css

```diff
@@ -87,6 +87,7 @@ html.rtl .attachment {
 	}
 
 	& .attachment-fields {
+		display: flex;
```

Comment:
this makes all not-short (long?) wrap in same row.

Reply:
yes, it does 😞 

@ggazzo will fix this